src/main/java/org/kpmp/repositoryDataset/RepositoryDatasetService.java

```diff
@@ -56,9 +62,19 @@ public RepositoryDatasetService(
 	}
 
   public List<RepositoryDataset> getRepositoryDataset() throws Exception {
-    List <RepositoryDataset> datasets = new ArrayList<>();
+    List <RepositoryDataset> datasets = new ArrayList<>(); 
     datasets.addAll(fileRepo.findAll());
-    return datasets;
+		String maxReleaseVersion = fileRepo.max();
```

Comment:
Your indentation here is pretty wild...let's line this up so it doesn't look like it is in a conditional or something.